### PR TITLE
DID major release changes

### PR DESCRIPTION
**Pull request by localredhead**

### CODE REVIEW

File: markdown/docs/guides/ReactUportTruffle.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ type: "content"
 source: "https://github.com/uport-project/uport-project.github.io/blob/develop/markdown/docs/guides/ReactUportTruffle.md"
 ---
 
+# TODO - Warning: React Truffle Box out of date!
```
Comment (zachferland) [inline markdown/docs/guides/ReactUportTruffle.md]: Added note here: need to update this file if ready in time (cc @rmw2 )

File: markdown/docs/guides/GettingStarted.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ type: "content"
 source: "https://github.com/uport-project/uport-project.github.io/blob/develop/markdown/docs/guides/GettingStarted.md"
 ---
 
+# TODO - Warning: Tutorial out of date!
```
Comment (zachferland) [inline markdown/docs/guides/GettingStarted.md]: Added note here: need to update this file for new libs, most concepts are similar, but should be even easier/clear to demonstrate this (cc @localredhead ). I could help update this as well

File: markdown/docs/guides/ReactUportTruffle.md (hunk 1 above)
Comment (localredhead) [reply markdown/docs/guides/ReactUportTruffle.md]: see #99

File: markdown/docs/guides/GettingStarted.md (hunk 2 above)
Comment (localredhead) [reply markdown/docs/guides/GettingStarted.md]: see #99

**Comment by localredhead** [comment]
closing as this work has migrated to #99

**Pull request closed without merging**